lib/rucio/core/replica.py

```diff
@@ -520,18 +551,27 @@ def get_bad_replicas_backlog(*, session: "Session"):
 
     :returns: a list of dictionary {'rse_id': cnt_bad_replicas}.
     """
-    query = session.query(func.count(models.RSEFileAssociation.rse_id), models.RSEFileAssociation.rse_id). \
-        with_hint(models.RSEFileAssociation, 'INDEX(DIDS DIDS_PK) USE_NL(DIDS) INDEX_RS_ASC(REPLICAS ("REPLICAS"."STATE"))', 'oracle'). \
-        filter(models.RSEFileAssociation.state == ReplicaState.BAD)
+    stmt = select(
+        func.count(models.BadReplicas.scope),
+        models.BadReplicas.rse_id
+    ).with_hint(
+        models.RSEFileAssociation, 'INDEX(DIDS DIDS_PK) USE_NL(DIDS) INDEX_RS_ASC(REPLICAS ("REPLICAS"."STATE"))', 'oracle'
+    ).where(
+        models.BadReplicas.state == BadFilesStatus.BAD
+    )
```

Comment:
The new query here seems so be slightly different compared to the old query (the queried columns are different and the `where` condition is different) - is this intended?